interpreter/python/python.go

```diff
@@ -803,11 +805,20 @@ func Loader(ebpf interpreter.EbpfHandler, info *interpreter.LoaderInfo) (interpr
 		vms.PyFrameObject.Code = 0
 		vms.PyFrameObject.LastI = 56       // _Py_CODEUNIT *prev_instr
 		vms.PyFrameObject.Back = 8         // struct _PyInterpreterFrame *previous
-		vms.PyFrameObject.EntryMember = 70 // char owner
+		vms.PyFrameObject.EntryMember = 70 // struct _PyInterpreterFrame owner
 		vms.PyFrameObject.EntryVal = 3     // enum _frameowner, FRAME_OWNED_BY_CSTACK
 		vms.PyThreadState.Frame = 56
 		vms.PyCFrame.CurrentFrame = 0
 		vms.PyASCIIObject.Data = 40
+	case pythonVer(3, 13):
+		vms.PyFrameObject.Code = 0
+		vms.PyFrameObject.LastI = 56       // _Py_CODEUNIT *prev_instr
+		vms.PyFrameObject.Back = 8         // struct _PyInterpreterFrame *previous
+		vms.PyFrameObject.EntryMember = 70 // struct _PyInterpreterFrame owner
+		vms.PyFrameObject.EntryVal = 3     // enum _frameowner, FRAME_OWNED_BY_CSTACK
+		vms.PyThreadState.Frame = 72
+		vms.PyCFrame.CurrentFrame = 8
+		vms.PyASCIIObject.Data = 40
```

Comment:
Seems this is mostly same as in 3.12. The difference being just in `PyThreadState.Frame` and `PyCFrame.CurrentFrame`.  But I suppose this is ok for now.